tests/cpp_check.sh

```diff
@@ -14,6 +14,7 @@ if command -v cppcheck > /dev/null; then
   u="$u -U __REDIRECT_NTH -U _BSD_RUNE_T_ -U _TYPE_size_t -U __LDBL_REDIR1_DECL"
   supp="--suppress=missingInclude --suppress=unusedFunction --suppress=noExplicitConstructor"
   supp="$supp --suppress=useInitializationList --suppress=uninitMemberVar"
+  supp="$supp --suppress=unknownMacro"
```

Comment:
Curious what these macros were. Sometimes you can make these "known" to cppcheck by including the correct path.

Reply:
It was generated by this
```
src/image.h:40:3: error: There is an unknown macro here somewhere. Configuration is required. If LIBCAMERA_DISABLE_COPY is a macro then please configure it. [unknownMacro]
  LIBCAMERA_DISABLE_COPY(Image)
  ^

```

Reply:
Thanks

Reply:
The real problem here is the header than defines this is in libcamera, if you run this step in the build after you install libcamera, this might just work, otherwise you can point this to libcamera include directory.

Reply:
The build of twincam should probably be first in general, then you know your environment is set up for everything else.

Reply:
This is more correctly fixed by adding `-I/usr/include/libcamera` to cppcheck. libcamera has a custom include path which you can see in meson.build